bigquery_datatransfer/nox.py

```diff
@@ -65,3 +95,18 @@ def system(session, py):
 
     # Run py.test against the system tests.
     session.run('py.test', '--quiet', 'tests/system/')
+
+
+@nox.session
+def cover(session):
+    """Run the final coverage report.
+
+    This outputs the coverage report aggregating coverage from the unit
+    test runs (not system test runs), and then erases coverage data.
+    """
+    session.interpreter = 'python3.6'
+    session.install('coverage', 'pytest-cov')
+    session.run('coverage', 'report', '--show-missing', '--fail-under=70')
+    # uncomment when coverage is met
+    # session.run('coverage', 'report', '--show-missing', '--fail-under=100')    
```

Comment:
@chemelnucfin I'm inclined to just check in a failing `nox -s cover` since there is no "automatic" way for us to follow-up on this when / if coverage gets higher. But maybe @lukesneeringer has other ideas.

Reply:
It would stall the builds, wouldn't it?

I think eventually someone will see that it's not high enough?

Reply:
How do you mean stall? It will fail the builds.

> eventually someone will see that it's not high enough

Can you elaborate? These config files are essentially set it and forget it (i.e. no one may ever look at that line again).

Reply:
That's what I mean, it will fail the builds until coverage gets to 100.  Which means we can't tell if other tests are successful or not.

It's not any worse than it is now.  If someone increases coverage, great.  If not, it's what we have.

Reply:
Can we leave a `cover` off for now and I will open a bug to correct in autogen (both the coverage itself and the nox item)?

Reply:
https://github.com/googleapis/toolkit/issues/1768

Reply:
removed.